leapp/snactor/__init__.py

```diff
@@ -60,6 +61,11 @@ def cli(args):
     os.environ['LEAPP_CONFIG'] = config_file_path
     os.environ['LEAPP_DEBUG'] = '1' if args.debug else os.environ.get('LEAPP_DEBUG', '0')
 
+    if os.environ['LEAPP_DEBUG'] == '1' or args.verbose:
```

Comment:
Oneliner is easier to read imho. 
iiuc that's the same functionality as in leapp.cli.__init__.py, any chance we could reuse code?

Reply:
I agree about one line, but again I need to do that way because of CI :)

Maybe we can open a topic in Cabal about the line limit.

It's a good idea to reuse this code, I will open a task in Jira about that because I have no more time in this sprint